src/ThisData.php

```diff
@@ -38,16 +46,19 @@ class ThisData
      */
     private static $endpointClassMap = [
         self::ENDPOINT_EVENTS => EventsEndpoint::class,
+        self::ENDPOINT_VERIFY => VerifyEndpoint::class
     ];
 
     /**
      * @param Client $client
      * @param RequestHandlerInterface $handler
+     * @param array|null $configuration
      */
-    public function __construct(Client $client, RequestHandlerInterface $handler)
+    public function __construct(Client $client, RequestHandlerInterface $handler, array $configuration = null)
```

Comment:
This configuration is endpoint specific isn't it? Would it make more sense to move this configuration variable to the `getVerifyEndpoint()` method?